### DISCUSSION TITLE
Add Panel#Icon

### DISCUSSION THREAD

**Discussion by HexedHero** [blish-hud/Blish-HUD#939] [Blish HUD Core Ideas]
Add an AsyncTexture2D to display an icon in the panel object before the text.
![](https://i.imgur.com/Wy58iWf.png)

**Comment by dlamkins** [comment]
There is actually a PR for this already. 🙂

**Comment by HexedHero** [comment]
Whoops, I checked issues and discussions but not PRs haha.
